lib/fluent/plugin/in_windows_eventlog2.rb

```diff
@@ -401,8 +404,8 @@ def parse_desc(record)
     end
 
     def to_key(key)
-      key.downcase!
-      key.gsub!(' '.freeze, '_'.freeze)
+      key = key.downcase! if @downcase_description_keys
```

Comment:
`String#downcase!` changes the self content.

```suggestion
      key.downcase! if @downcase_description_keys
```